install.yaml

```diff
@@ -144,7 +144,7 @@ post_install_actions:
     if [ ! -f ${DDEV_APPROOT}/.ddev/.env ]; then
       cp ${DDEV_APPROOT}/.ddev/scripts/sample.env ${DDEV_APPROOT}/.ddev/.env
       echo "Please configure upstream provider in .ddev/.env file!"
-    fi
```

Comment:
`if` is not closed!

Reply:
Not quite sure how that got removed. Best put it back in then.